src/components/expanded-state/ChartExpandedState.js

```diff
@@ -213,9 +76,9 @@ export default function ChartExpandedState({ asset }) {
               style: { flex: 1 },
             }}
           />
-        </ActionRow>
+        </SheetActionButtonRow>
       ) : (
-        <ActionRow key={`row${showChart}`}>
+        <SheetActionButtonRow key="actionRow">
```

Comment:
hey! is there a reason why we have `key`s on these components? (i know you didnt add them, only changed them, but curious why they are there at all)

Reply:
hey I think we can get rid of this! will update